src/components/Book.js

```diff
@@ -1,27 +1,58 @@
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { deleteBook } from '../redux/books/booksSlice';
+import './Book.css';
 
 const Book = ({ book }) => {
   const dispatch = useDispatch();
   const onRemoveClicked = (id) => {
     dispatch(deleteBook(id));
   };
   return (
-    <div>
-      <h1>{book.title}</h1>
-      <h2>
-        Author:
-        {book.author}
-      </h2>
-      <button onClick={() => onRemoveClicked(book.item_id)} type="button">Delete</button>
+    <div className="main-book" key={book.item_id}>
+      <div>
+        <p className="category">{book.category}</p>
+        <h1 className="Title">{book.title}</h1>
+        <p className="author">
+          Author:
+          {' '}
+          {book.author}
+        </p>
+        <div className="btn">
+          <button type="button" className="comments">Comments</button>
+          <div className="vertical" />
+          <button
+            type="button"
+            className="comments"
+            onClick={() => onRemoveClicked(book.item_id)}
+          >
+            Remove
+          </button>
+          <div className="vertical" />
+          <button type="button" className="comments">Edit</button>
+        </div>
+      </div>
+      <div className="circle">
+        <div className="oval" />
+        <div>
+          <h1 className="percentage">64%</h1>
+          <p className="Completed">Complete</p>
+        </div>
+      </div>
+      <div className="last">
+        <span className="Chapter">Current Chapter</span>
+        <span className="Lesson">Chapter 17</span>
+        <div className="update">Update Progress</div>
+      </div>
     </div>
```

Comment:
You're on the right track with your project, and I appreciate your attention to detail. However, there are some adjustments needed to align your card with the design more closely. Here are some areas to focus on: Background Color, Padding, Spacing Between Cards, Border Color, Gap with the Navbar,  Horizontal Alignment with the Navbar and background color.
| Your | Design |
| --- | --- |
|  ![image](https://github.com/BilalLiaquat7/bookstore/assets/55840999/47a23f2c-60e8-4114-9eb2-66c34883edfb)| ![image](https://github.com/BilalLiaquat7/bookstore/assets/55840999/999868eb-9332-42cf-a3af-7c5e88bf83fd)|

By making these adjustments, you'll bring your card in line with the design, creating a more visually consistent and cohesive user interface. Keep up the excellent work, and thank you for your dedication to achieving a design-compliant project! 🌟🎨